modules/javafx.controls/src/main/java/javafx/scene/control/TabPane.java

```diff
@@ -169,54 +169,34 @@ public TabPane(Tab... tabs) {
         return tabs;
     }
 
-    private ObjectProperty<SingleSelectionModel<Tab>> selectionModel = new SimpleObjectProperty<SingleSelectionModel<Tab>>(this, "selectionModel");
-
     /**
      * <p>Sets the model used for tab selection.  By changing the model you can alter
      * how the tabs are selected and which tabs are first or last.</p>
-     * @param value the selection model
      */
+    private ObjectProperty<SingleSelectionModel<Tab>> selectionModel = new SimpleObjectProperty<SingleSelectionModel<Tab>>(this, "selectionModel");
+
     public final void setSelectionModel(SingleSelectionModel<Tab> value) { selectionModel.set(value); }
 
-    /**
-     * <p>Gets the model used for tab selection.</p>
-     * @return the model used for tab selection
-     */
     public final SingleSelectionModel<Tab> getSelectionModel() { return selectionModel.get(); }
 
-    /**
-     * The selection model used for selecting tabs.
-     * @return selection model property
-     */
     public final ObjectProperty<SingleSelectionModel<Tab>> selectionModelProperty() { return selectionModel; }
 
-    private ObjectProperty<Side> side;
-
     /**
      * <p>The position to place the tabs in this TabPane. Whenever this changes
      * the TabPane will immediately update the location of the tabs to reflect
      * this.</p>
-     *
-     * @param value the side
+     * The default position for the tabs is {@code Side.Top}.
```

Comment:
Ordinarily we would use a `@defaultValue` tag here. In this case, you are just moving existing docs from one method to another, so it could be done in a follow-up bug. Can you file a follow-up bug?